libdirections-offboard/src/main/java/com/mapbox/navigation/route/offboard/RouteBuilderProvider.kt

```diff
@@ -27,7 +27,6 @@ internal object RouteBuilderProvider {
             .accessToken(accessToken)
             .voiceInstructions(true)
             .bannerInstructions(true)
-            .enableRefresh(false)
```

Comment:
This is a pain to find. I had to ask in #directions to figure it out
https://mapbox.slack.com/archives/C03KW1JQD/p1583881618015100

Reply:
Updating this feature to be enabled automatically with traffic profiles because that's the only place it's supported.
https://mapbox.slack.com/archives/C03KW1JQD/p1583883018020600?thread_ts=1583881618.015100&cid=C03KW1JQD